docker-compose.yml

```diff
@@ -14,6 +14,8 @@ services:
       timeout: 3s
       retries: 3
       start_period: 5s
+    ports:
+      - "5432:5432"
```

Comment:
Necessary?

Reply:
Not really, I was accessing the database from my editor directly instead of through docker